examples/conll2000_chunking_crf.py

```diff
@@ -0,0 +1,99 @@
+"""Train CRF and BiLSTM-CRF on CONLL2000 chunking data, similar to https://arxiv.org/pdf/1508.01991v1.pdf.
+"""
+from __future__ import absolute_import
+from __future__ import print_function
+
+import numpy
+from collections import Counter
+from sklearn.metrics import classification_report
```

Comment:
`sklearn` is not necessary, as I just use the `classification_report` to print the result in a nice way. `nltk` is here for getting the data. My  original idea was a bit on the opposite, avoiding having the same dataset in multiple packages or writing duplicate script to load data, but instead using an existing package, which also saves the effort of licensing.  But I understand the disadvantage as well. I'll change it, as it probably fits better to the Keras way.